app/scripts/controllers/registration.js

```diff
@@ -167,7 +167,7 @@ angular.module('confRegistrationWebApp')
       }), true);
     };
 
-    var findAnswersToSave = function(){
```

Comment:
This might have also motivated this change https://github.com/CruGlobal/conf-registration-web/pull/537/files/d78487aa8a65289b882e2b964439615581bd5291#diff-4d5aa81bf4f18104bb6ea53a8b5d1f43R11 so I didn't have to move this function to the top of the file. Changing that `latedef` config allows function definitions to be defined after their usages but still disallows function expressions to be defined after usage.